app/assets/javascripts/subjects.js.coffee

```diff
@@ -238,6 +287,29 @@ class window.FilterManager
       when "comment_status_unprocessed" then @getCommentAnalysisDisplayTextFromStatus val
       when "status" then @getImageAnalysisDisplayTextFromStatus val
       when "status_unprocessed" then @getImageAnalysisDisplayTextFromStatus val
+      when "vr1_entire_dataset" then "Entire dataset"
+      when "vr1_daytime" then "Daytime"
+      when "vr1_nighttime" then "Nighttime"
+      when "vr1_blank" then "Blank"
+      when "vr1_non_blank" then "Non-blank"
+      when "vr1_one_animal" then "One animal"
+      when "vr1_simple" then "Simple (2-5 animals"
+      when "vr1_complex" then "Complex (6+ animals"
+      when "vr1_single_species" then "Single species"
+      when "vr1_only_buffalo" then "Only buffalo"
+      when "vr1_only_elephant" then "Only elephants"
+      when "vr1_only_ostrich" then "Only ostriches"
+      when "vr1_only_warthog" then "Only warthogs"
+      when "vr1_only_wildebeest" then "Only wildebeest"
+      when "vr1_only_other" then "Only a single species, not listed above"
+      when "vr1_multi_species" then "Multiple species"
+      when "vr1_multi_including_buffalo" then "Multiple species, including buffalo"
+      when "vr1_multi_including_elephant" then "Multiple species, including elephants"
+      when "vr1_multi_including_ostrich" then "Multiple species, including ostriches"
+      when "vr1_multi_including_warthog" then "Multiple species, including warthogs"
+      when "vr1_multi_including_wildebeest" then "Multiple species, including wildebeest"
+      when "vr1_multi_including_none_of_the_five" then "Multiple species, including not listed above"
+      else query_field
```

Comment:
OK, i had originally liked the fact that any unknown param showed up as undefined, as it highlights the need for a dev fix. but I guess this is probably better in case someone types an unexpected but still supported param into the address bar. So on reflection, i approve of adding this!